src/PostgREST/Observation.hs

```diff
@@ -69,14 +66,12 @@ type ObservationHandler = Observation -> IO ()
 
 observationMessage :: Observation -> Text
 observationMessage = \case
-  AdminStartObs host port ->
-    "Admin server listening on " <> fromJust host <> ":" <> show (fromIntegral (fromJust port) :: Integer)
+  AdminStartObs address ->
+    "Admin server listening on " <> address
   AppStartObs ver ->
     "Starting PostgREST " <> T.decodeUtf8 ver <> "..."
-  AppServerPortObs host port ->
-    "API server listening on " <> host <> ":" <> show port
-  AppServerUnixObs sock ->
-    "API server listening on unix socket " <> show sock
```

Comment:
This "unix socket" message has changed right? How do we confirm the user knows the address is a unix socket?

Reply:
I think unix socket addresses are like this `/tmp/http.sock`. So the message we be like:
```
API server listening on /tmp/http.sock
```
So the user can infer from this that the socket is unix socket. We should have a test for this.